docs/monthly-meeting/2021-06-17.md

```diff
@@ -0,0 +1,74 @@
+# JupyterHub and BinderHub Team Meeting
+
+- **Date:** Thursday 17th June 2021
+- **Time:** 9AM Europe/London
+- **This HackMD:** <https://hackmd.io/@sgibson91/hubs-team-meeting>
+- **GitHub issue:** <https://github.com/jupyterhub/team-compass/issues/>
+- **Calendar for future meetings:** <https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings.html>
+
+#### By participating in this meeting, you are agreeing to abide by and uphold the [Code of Conduct](https://jupyter.org/conduct). Please take a second to read through it! :pray:
+
+## Welcome to the Team Meeting
+
+Hello!
+
+If you are joining the team video meeting, sign in below so we know who was here. Roll call:
+
+- Sarah Gibson / 2i2c / @sgibson91
+- Simon Li / University of Dundee / @manics
+- Tim Head / Binder / @betatim
+- Min Ragan-Kelley / Simula / @minrk
+- Callum Mole / Turing / @callummole
+- Ivana / - / @IvanaH8
+- Isabela Presedo-Floyd / Quansight Labs / @isabela-pf
+- Samuel Gaist / Idiap / @sgaist
+- Mridul / GESIS / @MridulS
+- Yuvi / UC Berkeley / 2i2c / @yuvipanda
+- Omar / Karlstad University / @0mar
+
+## Quick updates
+
+60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+-
+
+## Reports and celebrations
+
+This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+- Distinguished Jupyter contributors announced: https://blog.jupyter.org/congratulations-distinguished-contributors-149deff3033c
+- Erik & 1.0!
```

Comment:
Do you know what this is referring to? Z2JH 1.0.0?

Reply:
Yes I think so

Reply:
```suggestion
- Erik & Z2JH 1.0.0 release!
```

Reply:
Added a clarification :D